qemu/tests/driver_in_use.py

```diff
@@ -122,6 +122,7 @@ def run_bg_test_sep(sub_type):
     bg_stress_test = params["run_bgstress"]
     wait_time = float(params.get("wait_bg_time", 60))
     suppress_exception = params.get("suppress_exception", "no") == "yes"
+    wait_bg_finish = params.get("wait_bg_finish", "no")
```

Comment:
Should be `wait_bg_finish = params.get("wait_bg_finish", "no") == "yes"`

Reply:
modified,thanks